third_party/blink/public/platform/web_source_buffer.h

```diff
@@ -140,6 +144,11 @@ class WebSourceBuffer {
   // After this method is called, this WebSourceBuffer should never use the
   // client pointer passed to SetClient().
   virtual void RemovedFromMediaSource() = 0;
+
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  // Return the highest presentation timestamp written to the SbPlayer.
```

Comment:
We don't have to mention SbPlayer here, as the implementation is no longer SbPlayer specific.

The same to the commit message of the PR.